### PR TITLE
New gun sounds except base

### PR DESCRIPTION
**Pull request by pawnishoovy**
This is sounds for every faction HDFirearm (i.e. all of them except base)

**What this includes:**

Firing sounds
Reload sounds
Chambering sounds
Missile loop/starting sounds
Gatling gun spinning sounds
Credits for each sound

**What this does NOT include and is NOT represented properly in the example videos:**

Exploding sounds
Properly scripted/linked chambering sounds

**Misc info**

Most if not all firing sounds are stereo, this can/will be changed later as required, I didn't want to downmix them from the get-go.

COA Example video:

https://youtu.be/C2MyVgt2C7M

COA Example video 2: new shotgun, new sniper, squad firing

https://youtu.be/4tCVpupnQ_s

BRW Example video:

https://youtu.be/E73UVRuLfCg

BRW Example video 2: large scale combat

https://youtu.be/cfpJCabHCpw

**KNOWN ISSUES**

A lot of sounds have garbage metadata and need to be cleansed. They're not relevant in any way.

### CODE REVIEW

**Comment by MaximDude** [comment]
@fourZK @Envy-glitch can you guys start going through this?

File: Imperatus.rte/Devices/Weapons/Marauder/Marauder.ini [hunk 1]
```diff
@@ -167,16 +167,20 @@ AddDevice = HDFirearm
...
+			FilePath = Imperatus.rte/Devices/Weapons/Marauder/Sounds/Fire2.wav
+		AddSample = ContentFile
+			FilePath = Imperatus.rte/Devices/Weapons/Marauder/Sounds/Fire3.wav
 	EmptySound = Sound
 		AddSample = ContentFile
 			FilePath = Base.rte/Sounds/Devices/EmptyClick1.wav
 	ReloadStartSound = Sound
 		AddSample = ContentFile
-			FilePath = Base.rte/Sounds/Devices/ReloadStart.wav
+			FilePath = Imperatus.rte/Devices/Weapons/Marauder/Sounds/ReloadStart.wav
```
Comment (MaximDude) [inline Imperatus.rte/Devices/Weapons/Marauder/Marauder.ini]: I'm thinking this should have a chonky shell-by-shell reload.

File: Imperatus.rte/Devices/Weapons/Devastator/Devastator.ini [hunk 2]
```diff
@@ -231,16 +231,20 @@ AddDevice = HDFirearm
 		CopyOf = Muzzle Flash Shotgun
 	FireSound = Sound
 		AddSample = ContentFile
-			FilePath = Imperatus.rte/Devices/Weapons/Devastator/Sounds/DevastatorFire.wav
+			FilePath = Imperatus.rte/Devices/Weapons/Devastator/Sounds/Fire1.wav
```
Comment (MaximDude) [inline Imperatus.rte/Devices/Weapons/Devastator/Devastator.ini]: Not sure this fits what comes out of the barrel.

File: Ronin.rte/Devices/Weapons/M1Garand/M1Garand.ini [hunk 3]
```diff
@@ -17,9 +17,7 @@ AddEffect = AEmitter
 	JointStiffness = 1
 	BurstSound = Sound
 		AddSample = ContentFile
-			FilePath = Ronin.rte/Devices/Weapons/M1Garand/Sounds/GarandPing000.wav
-		AddSample = ContentFile
-			FilePath = Ronin.rte/Devices/Weapons/M1Garand/Sounds/GarandPing001.wav
+			FilePath = Ronin.rte/Devices/Weapons/M1Garand/Sounds/Ping.wav
```
Comment (MaximDude) [inline Ronin.rte/Devices/Weapons/M1Garand/M1Garand.ini]: Needs more than one ping sample. :judge:

File: Dummy.rte/Devices/Weapons/RailPistol/RailPistol.ini [hunk 4]
```diff
@@ -138,16 +138,20 @@ AddDevice = HDFirearm
 		CopyOf = Muzzle Flash Pistol
 	FireSound = Sound
 		AddSample = ContentFile
-			FilePath = Dummy.rte/Devices/Weapons/RailPistol/Sounds/RailPistolFire.wav
+			FilePath = Dummy.rte/Devices/Weapons/RailPistol/Sounds/Fire1.wav
```
[the remark this answers is not in the document]
Comment (pawnishoovy) [reply Dummy.rte/Devices/Weapons/RailPistol/RailPistol.ini]: it's not quite a pew pew pistol - test it out and see how much damage it does

File: Imperatus.rte/Devices/Weapons/Marauder/Marauder.ini (hunk 1 above)
Comment (pawnishoovy) [reply Imperatus.rte/Devices/Weapons/Marauder/Marauder.ini]: sprite looked like it had a weird unused magazine to me, i'll wait for further opinions

File: Ronin.rte/Devices/Weapons/M1Garand/M1Garand.ini (hunk 3 above)
Comment (pawnishoovy) [reply Ronin.rte/Devices/Weapons/M1Garand/M1Garand.ini]: i don't have any :(

File: Imperatus.rte/Devices/Weapons/Marauder/Marauder.ini (hunk 1 above)
Comment (fourZK) [reply Imperatus.rte/Devices/Weapons/Marauder/Marauder.ini]: I think the concept of firing hot chain is a bit unorthodox to consolidate a shell-by-shell reload.

**Comment by pawnishoovy** [comment]
> Here are my thoughts in one lump:

looked at everything i could except the dummies, i wanna keep those for last because they are painful

1. really nothing i can do for techion, i just don't have the raw volume and variety of sounds to find the right perfect stuff for everything. unless you literally just want me to add generic bass punches to everything

2. ronin looked at. how's the kar98 anywhere near farty though? maybe it sounds weird because it's missing the chamber

3. imperatus: toned down the chainy sounds and tried to shoddily unboom the shotty pistol, but i'll wait for more opinions on the rest

**Pull request merged**